docs/core/deploying/runtime-patch-selection.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Self-contained deployment runtime roll forward
+description: Learn about dotnet publish changes for self-contained deployments.
+author: jralexander
+ms.author: johalex
+ms.date: 05/30/2018
+---
+# Self-contained deployment runtime roll forward
+
+.NET Core [self-contained application deployments](index.md) include both the .NET Core libraries and the .NET Core runtime. Starting in .NET Core SDK 2.1.300 (.NET Core 2.1), a self-contained application deployment [now publishes the highest patch runtime on your machine](https://github.com/dotnet/designs/pull/36). By default,[`dotnet publish`](../tools/dotnet-publish.md) for a self-contained deployment selects the latest version installed as part of the SDK on the publishing machine. This enables your deployed application to run with security fixes (and other fixes) available during `publish`. The application must be re-published to obtain a new patch. Self-contained applications are created by specifying `--self-contained` on `dotnet publish` command or by specifying the RuntimeIdentifier in the project file or on the command line.
```

Comment:
Normally you wouldn't explicitly specify the `--self-contained` argument.  You need to specify a RID with `-r` anyway, and if you do, then `--self-contained` will default to true.  So I'd probably replace `--self-contained` with `-r <RID>` or similar in this sentence.

Reply:
Got it. Agreed